elements/lisk-dpos/src/delegates_info.ts

```diff
@@ -157,10 +158,26 @@ export class DelegatesInfo {
 				block.height + 1,
 				stateStore,
 			);
+
+			// Load 3 rounds of headers to calculate random seeds
+			const headersStartHeight = this.rounds.calcRoundStartHeight(
+				// tslint:disable-next-line:no-magic-numbers
+				round - 2,
+			);
+			const headersEndHeight = this.rounds.calcRoundEndHeight(round);
+			const headersForRandomSeeds = await this.chain.dataAccess.getBlockHeadersByHeightBetween(
```

Comment:
Prefer to use `stateStore.consensus.lastBlockHeaders` contains headers up to 309 blocks